src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -66,35 +66,11 @@ export const SystemCVEs = (props) => {
         }
     };
 
-    const apply = (config = {}) => {
-        if (Object.prototype.hasOwnProperty.call(config, 'cvss_filter')) {
-            let cvssEntry = CVSS_OPTIONS.find(item => item.value === config.cvss_filter);
-
-            if (typeof cvssEntry === 'undefined' && typeof config.cvss_filter !== 'undefined') {
-                let values = config.cvss_filter.split(/[^0-9]+/).filter(item => parseFloat(item));
-                cvssEntry = { from: values[0], to: values[1] };
-            } else if (typeof cvssEntry === 'undefined' && typeof config.cvss_filter === 'undefined') {
-                cvssEntry = { from: undefined, to: undefined };
-            }
-
-            config.cvss_from = cvssEntry.from;
-            config.cvss_to = cvssEntry.to;
-        }
-
-        if (Object.prototype.hasOwnProperty.call(config, 'publish_date')) {
-            let publicEntry = PUBLIC_DATE_OPTIONS.find(item => item.value === config.publish_date);
-
-            if (typeof publicEntry !== 'undefined') {
-                config.public_from = publicEntry.from && publicEntry.from.format('YYYY-MM-DD');
-                config.public_to = publicEntry.to && publicEntry.to.format('YYYY-MM-DD');
-            } else {
-                config.public_from = undefined;
-                config.public_to = undefined;
-                config.publish_date = undefined;
-            }
-        }
+    const apply = (filterParams = {}) => {
+        const params = constructFilterParameters(filterParams);
 
-        dispatch(changeSystemCVEsParameters(config));
+        createUrlParams(params);
```

Comment:
Just as a reminder, on solving conflicts please delete this line. Because of the  ([previous PR](https://github.com/RedHatInsights/vulnerability-ui/commit/d9204ec7437d4be43f8b3e133b24e2a06b00cc67#diff-39f36c104c5b28425391b5157f0774edR108))

Reply:
I believe now it's the same as the PR you mentioned

Reply:
there is not createUrlParams function call inside appy function